datasets/huggingface-c4/metadata.json

```diff
@@ -37,6 +37,7 @@
   "description": "A colossal, cleaned version of Common Crawl's web crawl corpus.\n\nBased on Common Crawl dataset: \"https://commoncrawl.org\".\n\nThis is the processed version of Google's C4 dataset by AllenAI.\n",
   "citation": "\n@article{2019t5,\n    author = {Colin Raffel and Noam Shazeer and Adam Roberts and Katherine Lee and Sharan Narang and Michael Matena and Yanqi Zhou and Wei Li and Peter J. Liu},\n    title = {Exploring the Limits of Transfer Learning with a Unified Text-to-Text Transformer},\n    journal = {arXiv e-prints},\n    year = {2019},\n    archivePrefix = {arXiv},\n    eprint = {1910.10683},\n}\n",
   "url": "https://huggingface.co/datasets/c4",
+  "version": "0.0.0",
```

Comment:
Should this be `"0.0"` or is the patch intended here?

Reply:
If we import from HF, then the versioning that they use is indeed `MAJOR.MINOR.PATCH` so I am maintaining it to avoid confusion